web/forms.py

```
@@ -802,20 +802,26 @@
-    Validates that video URLs are from YouTube or Vimeo with proper video ID formats.
+    Validates that video URLs are from YouTube or Vimeo.
     """
 
+    # Make description optional in the form
+    description = forms.CharField(
+        required=False,
+        widget=TailwindTextarea(
+            attrs={
+                "rows": 4,
+                "placeholder": "Describe what viewers will learn from this video (optional)",
+            }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)